[BUG] Marlin compile failed at PID DEBUG

<!--

Please follow the instructions below. Failure to do so may result in your issue being closed.

### Before Reporting a Bug

1. Test with the `bugfix-2.0.x` branch to see whether the issue still exists.

2. Get troubleshooting help from the Marlin community to confirm it's a bug and not just a configuration error. Links at https://github.com/MarlinFirmware/Marlin/issues/new/choose

### Instructions

1. Fill out every section of the template below.

2. Always attach configuration files, regardless of whether you think they are involved.

3. Read and understand Marlin's Code of Conduct. By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/MarlinFirmware/Marlin/blob/master/.github/code_of_conduct.md

-->

### Bug Description


Compiling .pio\build\BIGTREE_GTR_V1_0\src\src\gcode\temp\M303.cpp.o
Compiling .pio\build\BIGTREE_GTR_V1_0\src\src\gcode\units\M82_M83.cpp.o
Compiling .pio\build\BIGTREE_GTR_V1_0\src\src\lcd\dogm\lcdprint_u8g.cpp.o
Compiling .pio\build\BIGTREE_GTR_V1_0\src\src\lcd\dogm\marlinui_DOGM.cpp.o
Marlin\src\gcode\temp\M303.cpp: In static member function 'static void GcodeSuite::M303()':
Marlin\src\gcode\temp\M303.cpp:54:27: error: 'pid_debug_flag' was not declared in this scope; did you mean 'marlin_debug_flags'?
   54 |       serialprintln_onoff(pid_debug_flag);
      |                           ^~~~~~~~~~~~~~
      |                           marlin_debug_flags
*** [.pio\build\BIGTREE_GTR_V1_0\src\src\gcode\temp\M303.cpp.o] Error 1
==================================================================================== [FAILED] Took 150.06 seconds ====================================================================================

Environment       Status    Duration
----------------  --------  ------------
BIGTREE_GTR_V1_0  FAILED    00:02:30.060
================================================================================ 1 failed, 0 succeeded in 00:02:30.060 ================================================================================
The terminal process "C:\Users\Acer\.platformio\penv\Scripts\platformio.exe 'run'" terminated with exit code: 1.

Terminal will be reused by tasks, press any key to close it.

> Executing task: C:\Users\Acer\.platformio\penv\Scripts\platformio.exe run <


Configuration Files

[Marlin.zip](https://github.com/MarlinFirmware/Marlin/files/6097367/Marlin.zip)

--- a/Marlin/src/gcode/temp/M303.cpp
+++ b/Marlin/src/gcode/temp/M303.cpp
@@ -51,7 +51,7 @@ void GcodeSuite::M303() {
       thermalManager.pid_debug_flag ^= true;
       SERIAL_ECHO_START();
       SERIAL_ECHOPGM("PID Debug ");
-      serialprintln_onoff(pid_debug_flag);
+      serialprintln_onoff(thermalManager.pid_debug_flag);
       return;
     }
   #endif

